SECURITY.md

```diff
@@ -12,17 +12,17 @@ Minor versions of the Tanzu CLI are expected to be released every three months.
 
 Security is of the highest importance and all security vulnerabilities or suspected security vulnerabilities should be reported to Tanzu CLI privately, to minimize attacks against current users of Tanzu CLI before they are fixed. Vulnerabilities will be investigated and patched on the next patch (or minor) release as soon as possible. This information could be kept entirely internal to the project.
 
-If you know of a publicly disclosed security vulnerability for Tanzu CLI, please **IMMEDIATELY** contact the VMware Security Team (security@vmware.com). The use of encrypted email is encouraged. The public PGP key can be found [here](https://knowledge.broadcom.com/external/article?legacyId=1055). **IMPORTANT: Do not file public issues on GitHub for security vulnerabilities**
+If you know of a publicly disclosed security vulnerability for Tanzu CLI, please **IMMEDIATELY** contact the Broadcom Security Team (`vmware.psirt@broadcom.com`). The use of encrypted email is encouraged. The public PGP key can be found [here](https://knowledge.broadcom.com/external/article?legacyId=1055). **IMPORTANT: Do not file public issues on GitHub for security vulnerabilities**
```

Comment:
Why use forward quotes?  I assume it is to disable the hyperlink?  Although I don't use the `mailto:` link myself, could it not be useful for some people?

Reply:
Sorry, I wasn't clear.  If you just remove the forward quotes, the markdown automatically adds the `mailTo`.
This is the current Security.md result.
![Screenshot 2024-06-19 at 06 56 10](https://github.com/vmware-tanzu/tanzu-cli/assets/414402/62a2325c-b577-4868-bde0-f06763400daa)

So what I'm suggesting is just changing the email, nothing else.  That way users can click on the link, or they can easily copy/paste it since it is visible.

Reply:
Got it.